services/entry.js

```diff
@@ -0,0 +1,14 @@
+const { Entry } = require('../models')
+
+exports.getEntries = async () => {
+  try {
+    const entries = await Entry.findAll({
+      attributes: {
+        exclude: ['updatedAt', 'deletedAt'],
+      },
```

Comment:
Gabi, as was stated on the PR descritpion, you should only get entries that are only of the "news" type. On the same table we will have other types of entries such as: "testimonials".